Wearable/src/main/java/com/vorsk/minimalin/watchface/MinimalinWatchFaceService.java

```diff
@@ -1018,6 +1026,8 @@ public void onVisibilityChanged(boolean visible) {
 
                 // Preferences might have changed since last time watch face was visible.
                 loadSavedPreferences();
+                // need to update text size (is there a better way?)
```

Comment:
Why not put the code to set the text size here in `onVisibilityChanged` instead of  calling `initializeWatchFace` so we don't need to initialize everything?